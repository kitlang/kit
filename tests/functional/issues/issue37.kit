using rules MyRules;
using implicit ("a", "b", "c");

rules MyRules {
    (SOMETHING) => printf("it worked\n");
}

function testFunc(x: (CString, CString, CString)) {
    printf("%s %s %s\n", x[0], x[1], x[2]);
}

function main() {
    testFunc();
    SOMETHING;
}
