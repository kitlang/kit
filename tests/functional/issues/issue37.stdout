a b c
it worked
